SensusUI/Inputs/NumberSliderInput.cs

```diff
@@ -172,48 +240,69 @@ public override View GetView(int index)
                     {
                         _incrementalValue = newIncrementalValue;
                         _incrementalValueHasChanged = true;
-                        _sliderLabel.Text = GetLabelText(index) + "  " + _incrementalValue;
+                        _sliderLabel.Text = DisplaySliderValue ? GetLabelText(index) + "  " + _incrementalValue : GetLabelText(index) + "  " + _tipText;
                         Complete = Value != null;
                     }
                 };
 
-                base.SetView(new StackLayout
-                    {
-                        Orientation = StackOrientation.Vertical,
-                        VerticalOptions = LayoutOptions.Start,
-                        Children =
-                        { 
-                            _sliderLabel,
-                            new StackLayout
+                StackLayout stack = new StackLayout();
+                stack.Orientation = StackOrientation.Vertical;
+                stack.VerticalOptions = LayoutOptions.Start;
+                stack.Children.Add(_sliderLabel);
```

Comment:
Check out lines 180-187 in the code window above for the syntax. I'm not quite sure what you mean by the four combinations. Just give it a try and let's see how it looks.